backend/zfs_operations/services/pool_service.py

```
@@ -0,0 +1,815 @@
+            if not result.is_success:
+                return Result.failure(PoolException(
+                    f"Failed to get I/O statistics: {result.stderr}",
+                    error_code="IOSTAT_FAILED"
+                ))
+            
+            # Parse iostat output
+            iostat_data = await self._parse_iostat_output(result.stdout)
+            if iostat_data.is_failure:
+                return Result.failure(iostat_data.error)
+            
+            self._logger.info(f"Successfully retrieved I/O statistics")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)